WcaOnRails/app/models/continent.rb

```diff
@@ -11,11 +11,11 @@ def self.country_ids(continent_id)
   end
 
   def name
-    I18n.t(recordName, scope: :continents)
+    I18n.t(read_attribute(:name), scope: :continents)
```

Comment:
@jfly spaces are allows in both YAML keys and JS obiect keys, so I wouldn't expect any issues =)